### PR TITLE
Enable renovate automerge

### LINKED ISSUE #144: Configure Renovate auto-merge for "non-major-dev-dependencies"

### PR DESCRIPTION
**Pull request by aquarat**
Closes #144

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -3,26 +3,28 @@
   "extends": ["config:base"],
   "packageRules": [
     {
-      "matchPackageNames": ["node", "@octokit/rest"],
+      "matchPackageNames": ["node", "is-wsl", "@octokit/rest"],
```
Comment (Siegrift) [inline renovate.json:6]: Commons doesn't use `is-wsl`

**Review by Siegrift — CHANGES_REQUESTED** [review]

File: renovate.json [hunk 2]
```diff
@@ -3,26 +3,28 @@
...
+      "groupName": "non-major-dependencies",
+      "automerge": true
     }
   ],
   "rangeStrategy": "bump",
   "lockFileMaintenance": {
     "enabled": false
   },
-  "reviewers": ["Siegrift"],
+  "reviewers": ["team:commons-core"],
```
[the remark this answers is not in the document]
Comment (aquarat) [reply renovate.json:28]: I created that team 😅 , but I'll remove it 👌

File: renovate.json (hunk 1 above)
Comment (aquarat) [reply renovate.json:6]: 👍

**Review by Siegrift — APPROVED** [review]
👍 LGTM

**Pull request merged**